src/icon-buttons/icon-button.component.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import useAddButtonStyling from './use-add-button-styling.component'
+
+export default function IconButton(props) {
+  useAddButtonStyling()
```

Comment:
this does result in a bit of flickering as the UE is trying to get styling on the page...maybe it should just be a fn

Reply:
I think we should just use rollup-plugin-postcss, no? It automatically creates style tag and then appends to dom when you import it? And the `<style>` element is inline - no network request or webpack public path needed. Right?

Reply:
you got it